sirepo/sim_data/flash.py

```diff
@@ -5,15 +5,17 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
+from pykern import pkio
 import sirepo.sim_data
 import sirepo.util
 
 
 
 class SimData(sirepo.sim_data.SimDataBase):
 
-    EXE_NAME = 'flash4'
-    SETUP_UNITS_FILE = 'setup_units'
+    FLASH_RPM = 'flash.rpm'
```

Comment:
remove FLASH_RPM

Reply:
It's been removed. The code you're looking at is outdated.